src/lib/rule-finder.js

```diff
@@ -72,13 +72,16 @@ function _getPluginRules(config) {
 function _getAllAvailableRules(pluginRules) {
   var allRules = fs
     .readdirSync('./node_modules/eslint/lib/rules')
-    .map(function removeJsFromFilename(filename) {
-      return filename.replace(/\.js$/, '')
-    })
+  var filteredAllRules = []
+  allRules.forEach(function filterAllRules(filename) {
```

Comment:
Curious to know why do you think `filter + map` would be better in these kind of scenario?